dll/cpl/appwiz/createlink.c

```diff
@@ -447,8 +409,33 @@ FinishDlgProc(HWND hwndDlg,
             pContext = (PCREATE_LINK_CONTEXT) GetWindowLongPtr(hwndDlg, DWLP_USER);
             if (lppsn->hdr.code == PSN_SETACTIVE)
             {
-                /* TODO: Use shell32!PathCleanupSpec instead of DoConvertNameForFileSystem */
```

Comment:
Why removing the TODO? Has it turned out to be an invalid recommendation?

Reply:
`$ git blame dll/cpl/appwiz/createlink.c`:
![image](https://github.com/reactos/reactos/assets/2107452/dc12a5eb-8ea1-434b-bc86-c6102056fd8c)
`Katayama Hirofumi MZ 2020-01-29`

Reply:
Added call of `PathCleanupSpec`.